definition/data/sheets/AuthorisationCaseEvent.json

```diff
@@ -212,7 +212,7 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "RejectOrganisationPaymentPlan", "UserRole": "caseworker-cmc-anonymouscitizen", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "AgreementRejectedByDefendant", "UserRole": "caseworker-cmc-anonymouscitizen", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "AgreementCounterSignedByDefendant", "UserRole": "caseworker-cmc-anonymouscitizen", "CRUD": "CR"},
-  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "AgreementSignedByBoth", "UserRole": "caseworker-cmc-anonymouscitizen", "CRUD": "CR"},
```

Comment:
this should also be CR if there is an existing process for it in SupportController

Reply:
Agreed.

Reply:
Yeah it looks like the auth should be `CR` for all roles

Reply:
Code review comment addressed.